MediaElch.pro

```diff
@@ -29,8 +27,25 @@ win32 {
     DEFINES+=_UNICODE
 }
 
-TARGET = MediaElch
-TEMPLATE = app
+gcc | clang {
+    # Include all Qt modules using isystem so that warnings reagarding Qt files are ignored
+    QMAKE_CXXFLAGS += -isystem "$$[QT_INSTALL_HEADERS]"
+    for (inc, QT) {
+        QMAKE_CXXFLAGS += -isystem \"$$[QT_INSTALL_HEADERS]/Qt$$system("echo $$inc | sed 's/.*/\u&/'")\"
+    }
+}
+
+gcc {
```

Comment:
Just found out by using travis-ci, that these warnings are set even if the compiler is clang. I have to fix this...